ChangeLog

```diff
@@ -5,6 +5,8 @@ mbed TLS ChangeLog (Sorted per branch, date)
 Features
    * Add support for ARIA cipher (RFC 5794) and associated TLS ciphersuites
      (RFC 6209). Disabled by default, see MBEDTLS_ARIA_C in config.h
+   * Add additional block mode, OFB (Output Feedback), to the AES module and
+     cipher abstraction module.
```

Comment:
Whoops, this one is easy to miss. This is adding a feature to 2.10.0, which isn't really a good idea.

Reply:
This can be fixed in gatekeeping and merging.